prime-router/src/main/kotlin/metadata/DatabaseLookupTable.kt

```diff
@@ -0,0 +1,56 @@
+package gov.cdc.prime.router.metadata
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonMapperBuilder
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.google.common.base.Preconditions
+import gov.cdc.prime.router.azure.DatabaseLookupTableAccess
+import org.apache.logging.log4j.kotlin.Logging
+import org.jooq.exception.DataAccessException
+
+/**
+ * Database based lookup table.
+ * @param name the name of the table
+ * @param tableDbAccess for dependency injection
+ */
+class DatabaseLookupTable(
+    val name: String,
+    private val tableDbAccess: DatabaseLookupTableAccess = DatabaseLookupTableAccess()
+) : LookupTable(emptyList()), Logging {
```

Comment:
Rather than inherit from LookupTable what's the common interface they could both implement?

Reply:
I am not implementing new table operations, plus the LookTable does not have a load.  Even if I create an interface class I would still need to inherit from LookupTable.